TOC.json

```diff
@@ -22,38 +24,101 @@
     "documents": [
         {
             "name": "PlayFabAdminApi",
+            "pfurl": "swagger/Admin.swagger.json",
```

Comment:
Why don't you go through and make ALL of the swagger documents optional for now.
That way fetch won't fail in the interim period before these are available from MainServer.